scripts-available/CDB_UserTables.sql

```diff
@@ -6,34 +6,23 @@
 -- Currently accepted permissions are: 'public', 'private' or 'all'
 --
 CREATE OR REPLACE FUNCTION CDB_UserTables(perm text DEFAULT 'all')
-RETURNS SETOF information_schema.sql_identifier
+RETURNS SETOF name
```

Comment:
This shouldn't return a `SETOF name` cause that type is limited to 63 chars and schema + table_name can exceed that limit.

How about using `regclass` that has automatic casting to text types?